test/integ_tests/requirements.txt

```diff
@@ -1,19 +0,0 @@
-amazon-braket-sdk
```

Comment:
why do we no longer need this file? Are we just merging this with the requirements file for NBIs?

Reply:
Yes, this was to address the discussed proposal around not installing unpinned dependencies and instead using just the requirements dependencies.